packages/static-build/test/fixtures/astro-v4/probes.json

```diff
@@ -0,0 +1,5 @@
+{
+  "probes": [
+    { "path": "/", "mustContain": "Welcome to Astro" }
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Is there a reasonable way to test the "Incremental Content Caching" feature too?